### ISSUE TITLE
More prominently warn the user if there's no controller connected

### ISSUE THREAD

**Issue by hikari-no-yume** [hikari-no-yume/touchHLE#12]
I think it's easy to miss the current warning that's logged in the terminal. We could display it in a more obvious, GUI way. Not completely sure about this since it could be quite annoying though.

[`sdl2::messagebox`](https://docs.rs/sdl2/0.35.1/sdl2/messagebox/index.html) would be an easy way to do it, but I think people who don't have their controller connected all the time might get frustrated at having to dismiss the prompt.

Maybe ImGui would be good for this?

**Comment by Calinou** [comment]
> [sdl2::messagebox](https://docs.rs/sdl2/0.35.1/sdl2/messagebox/index.html) would be an easy way to do it, but I think people who don't have their controller connected all the time might get frustrated at having to dismiss the prompt.

Can you display a notification toast using SDL2? This is a good way to provide a temporary, non-blocking message.

**Comment by hikari-no-yume** [comment]
I don't think so, it's not a feature games tend to need and therefore not something SDL uses.
